ch_tools/chadmin/cli/table_group.py

```diff
@@ -242,20 +252,51 @@ def columns_command(ctx, database_name, table_name):
     default=False,
     help="Enable dry run mode and do not perform any modifying actions.",
 )
+@option(
+    "--detached",
+    is_flag=True,
+    help="Delete detached tables (with nonreplicated engine).",
+)
+@constraint(
+    If(IsSet("detached"), then=require_all),
+    [
+        "database_name",
+        "table_name",
+        "sync_mode",
```

Comment:
Shouldn't it be in `accept_none` group?

Reply:
No, sync_mode has default=True and remove works sync. There is no support for async drop detached table.